docs/articles_en/documentation/openvino-ir-format/operation-sets/operations-specifications/movement/strided-slice-1.rst

```diff
@@ -14,6 +14,69 @@ StridedSlice
 
 **Short description**: *StridedSlice* extracts a strided slice of a tensor.
 
+**Detailed description**: The *StridedSlice* operation extracts a slice from a given tensor based on computed indices from the inputs: begin (inclusive of the element at the given index), end (exclusive of the element at the given index), and stride, for each dimension.
+
+The operation takes inputs with the following properties:
+
+* :math:`input` tensor to slice, with N dimensions.
+* :math:`begin, end, stride` inputs - 1D lists of integers of the same length M. **Stride input cannot contain any zeros.**
+* :math:`begin\_mask, end\_mask, new\_axis\_mask, shrink\_axis\_mask, ellipsis\_mask inputs` - bitmasks, 1D lists of integers (0 or 1). **Each mask can have a unique length**. :math:`ellipsis\_mask` can have up to one occurrence of the value 1.
+* :math:`new\_axis\_mask, shrink\_axis\_mask, ellipsis\_mask` are used to modify the output dimensionality of the data. If they are unused, N == M. Otherwise, N does not necessarily equal M.
+
+The basic slicing operation accumulates output elements as follows:
+
+* The operation iterates over the values of begin, end, and stride. At every step, the operation uses the i-th element of begin, end, and stride to perform the slicing.
+* Let :math:`slicing\_index = begin[i]`. This value determines the first index to start slicing. This sliced element is added to the output.
+* If :math:`begin[i] == end[i]`, only a single element is added to the output. The corresponding output dimension is then equal to 1 (in other words, the dimension is kept).
```

Comment:
This case explains the behavior of selecting a single element from a corresponding dimension, not necessarily the last. 

To explain it simply, assume a Python numpy array of shape = [2, 3, 2] with elements [ [[1 2] [3 4] [5 6]]  [[7 8] [9 10] [11 12]] ]. Performing the operation array2 = array[:, 1, :] results in array2 with shape = [2, 1, 2], where for the second dimension, only the element with ID 1 is taken, so the array2 values are [ [[3 4]], [[9,10]] ]. This slicing (array[:, 1, :]) is equivalent to writing array[:, 1:2, :] as well.

Reply:
In my mind, an element is a value from tensor(blob, data) but, in you mind, an element is dimension of shape 😄
So I read `If begin[i] == end[i], only a single element is added to the output.` like `only one element of tensor (data, blob) will be copied`. This definition is valid for last dimension slicing.
Maybe we should consider here smth like `only a single element of the corresponding dimension is added to the output` because it's not clear - what is the element.
What do you think?

Reply:
Sounds good, added!